package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "lint": "eslint .",
     "test": "./test/e2e/compare.sh",
+    "test-package": "node --test ./test/package-tests/**.test.js",
```

Comment:
if you create a new npm script, it won't be run on CI. instead, please turn the existing `test` script into one which runs both the existing compare script (which you can rename to e.g. `test:compare`) and the new one you introduced, which for consistency you can name e.g. `test:package`. The `test` script can then simply become: `npm run test:compare && npm run test:package`

Reply:
Good suggestion, implemented 🙇